app/config/FrontendAppConfig.scala

```diff
@@ -71,4 +72,15 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   lazy val apiUrl = configuration.get[Service]("microservice.services.common-transit-convention-traders").baseUrl
 
   lazy val declarationEnabled: Boolean = configuration.get[Boolean]("declaration.enabled")
+
+  def traderDetailsFrontendUrl(lrn: LocalReferenceNumber): String    = frontendUrl(lrn, "traderDetails")
+  def routeDetailsFrontendUrl(lrn: LocalReferenceNumber): String     = frontendUrl(lrn, "routeDetails")
+  def transportDetailsFrontendUrl(lrn: LocalReferenceNumber): String = frontendUrl(lrn, "transportDetails")
+  def guaranteeDetailsFrontendUrl(lrn: LocalReferenceNumber): String = frontendUrl(lrn, "guaranteeDetails")
+  def itemsFrontendUrl(lrn: LocalReferenceNumber): String            = frontendUrl(lrn, "items")
+
+  private def frontendUrl(lrn: LocalReferenceNumber, section: String): String = {
+    val url: String = configuration.get[String](s"urls.${section}Frontend")
+    url.replace(":lrn", lrn.toString)
```

Comment:
We could maybe make something along the lines of what we do for service url's to avoid replacing values e.g. https://github.com/hmrc/manage-transit-movements-departure-frontend/blob/main/app/config/Service.scala

Reply:
Ah possibly, but would we still not run into the same issue of needing a `:lrn` placeholder in the startUrl?